unumpy/tests/test_numpy.py

```diff
@@ -300,6 +320,11 @@ def test_array_creation(backend, method, args, kwargs):
             if backend in FULLY_TESTED_BACKENDS and (backend, method) not in EXCEPTIONS:
                 raise
             pytest.xfail(reason="The backend has no implementation for this ufunc.")
+        except TypeError:
+            if method is np.asanyarray:
+                raise pytest.xfail(
+                    reason="The ufunc for this backend got an unexpected keyword."
+                )
```

Comment:
If not marking as `xfail`, you should reraise the error to improve the test output.

Reply:
By re-raise, I mean just a bare raise:
```python
if method is np.asanyarray:
    raise pytest.xfail()
else:
    raise
```

That will propagate the error instead of hiding it.

Reply:
Sorry, wasn't totally sure about what you meant. I'll fix it.